src/TerminalSettings/TerminalSettings/MainPage.cpp

```diff
@@ -14,6 +14,11 @@ namespace winrt::TerminalSettings::implementation
     MainPage::MainPage()
     {
         InitializeComponent();
+
+        // TODO: When we actually connect this to Windows Terminal,

```

Comment:
You mean read/write the settings json file?

Reply:
Sorta? Like, WT has some objects in `TerminalSettings` (that's what I'm working on now). I'll replace these "fake" objects with the real ones when they're done :)